expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts

```diff
@@ -6,7 +6,7 @@ type ScreenWidth = {
   screenWidth: number | undefined;
 };
 
-export const MainContainer = styled.View<ScreenWidth>`
+export const MainContainer = styled.ScrollView<ScreenWidth>`
   width: ${({ screenWidth }) => (screenWidth > breakpoints.tablet ? '80%' : '100%')};
   margin-horizontal: auto;
   margin-top: 40px;
```

Comment:
```suggestion
  padding-top: 32px;
```

Reply:
The padding adds to the element's width or height, so I am wondering why padding?

Reply:
With Margin-top u can notice the it added a space that doesn't scroll alone with the content, making it appear weird

https://user-images.githubusercontent.com/36667083/228395061-0bdf16cd-8450-438d-9632-1bdc87b0a6cd.mp4

with padding-top, the space is still there but scrolls with the contents


https://user-images.githubusercontent.com/36667083/228395214-9570fb95-3eca-440d-a918-775017ac164c.mp4